### ISSUE TITLE
is it possible to predict the turning point of time series?

### ISSUE THREAD

**Issue by yuchengzh** [oliverguhr/transformer-time-series-prediction#30]
thanks for your great work! 
I have tried your single-step script to train and test for a certain time series dataset.
However, I noticed that although the predicted curve is close to the ground truth by mean squared error, it seems always falls behind in predicting turning point of the curve.
For example, if the actual turning point appears at time point 10, my predicted turning point will probably appear at time point 11 or 12, seems unable to predict that turning point at the actual time point.
did you run into the same issue by any chance? do you have any suggestions?

**Comment by oliverguhr** [comment]
Interesting question, I see multiple options here:

- train with more data
- add turning point accuracy to the loss function (basically give bonus points for predicting the turning point correctly)
- try pretrained models like [timesfm](https://huggingface.co/google/timesfm-1.0-200m) 

Let me know what worked for you :)

Best, 
Oliver
